src/test/scala/io/github/spark_redshift_community/spark/redshift/UtilsSuite.scala

```diff
@@ -73,4 +79,16 @@ class UtilsSuite extends FunSuite with Matchers {
     assert(Utils.getRegionForRedshiftCluster("mycluster.example.com") === None)
     assert(Utils.getRegionForRedshiftCluster(redshiftUrl) === Some("us-west-2"))
   }
+
+  test("checkThatBucketHasObjectLifecycleConfiguration with no prefix") {
```

Comment:
Thanks for adding a test! It seems that coverage decreased. Could you please parametrize this to 'cover' all four branches ?  
1. ENABLED
2. key starts with prefix
3. rule prefix is null (just in case we keep this condition)
4. and the `false` case

Reply:
yes I will do that.